test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -180,25 +180,23 @@ void Interpreter::operator()(Block const& _block)
 
 u256 Interpreter::evaluate(Expression const& _expression)
 {
-	ExpressionEvaluator ev(m_state, m_dialect, m_variables, m_functions, m_scopes);
+	ExpressionEvaluator ev(m_state, m_dialect, m_variables, m_scopes);
 	ev.visit(_expression);
 	return ev.value();
 }
 
 vector<u256> Interpreter::evaluateMulti(Expression const& _expression)
 {
-	ExpressionEvaluator ev(m_state, m_dialect, m_variables, m_functions, m_scopes);
+	ExpressionEvaluator ev(m_state, m_dialect, m_variables, m_scopes);
 	ev.visit(_expression);
 	return ev.values();
 }
 
 void Interpreter::closeScope()
 {
-	for (auto const& var: m_scopes.back())
-	{
-		size_t erased = m_variables.erase(var) + m_functions.erase(var);
-		solAssert(erased == 1, "");
-	}
+	for (auto const& [var, funDeclaration]: m_scopes.back())
+		if (!funDeclaration)
+			solAssert(m_variables.erase(var) == 1, "");
```

Comment:
```suggestion
			solAssert(funDeclaration || m_variables.erase(var) == 1, "");
```

Reply:
I don't think this is needed because of `if (!funDeclaration)` in the line above.